pkg/scheduler/plugins/subgrouporder/subgroup_order.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package subgrouporder
+
+import (
+	"github.com/NVIDIA/KAI-scheduler/pkg/scheduler/api/podgroup_info"
+	"github.com/NVIDIA/KAI-scheduler/pkg/scheduler/framework"
+)
+
+type subGroupOrderPlugin struct {
+	// Arguments given for the plugin
+	pluginArguments map[string]string
+}
+
+func New(arguments map[string]string) framework.Plugin {
+	return &subGroupOrderPlugin{pluginArguments: arguments}
+}
+
+func (sgop *subGroupOrderPlugin) Name() string {
+	return "subgrouporder"
+}
+
+func (sgop *subGroupOrderPlugin) OnSessionOpen(ssn *framework.Session) {
+	ssn.AddSubGroupsOrderFn(SubGroupOrderFn)
+}
+
+func SubGroupOrderFn(l, r interface{}) int {
+	lv := l.(*podgroup_info.SubGroupInfo)
+	rv := r.(*podgroup_info.SubGroupInfo)
+
+	lNumActiveTasks := lv.GetNumActiveAllocatedTasks()
+	rNumActiveTasks := rv.GetNumActiveAllocatedTasks()
+
+	// Prioritize SubGroup below minAvailable
+	lGangSatisfied := lNumActiveTasks >= int(lv.MinAvailable)
+	rGangSatisfied := rNumActiveTasks >= int(rv.MinAvailable)
+	if !lGangSatisfied && !rGangSatisfied {
+		return 0
```

Comment:
Let's use consts of lSubGroupPrioritized/rSubGroupPrioritized like we do for queues:
```Go
const (
	lQueuePrioritized   = -1
	rQueuePrioritized   = 1
	equalPrioritization = 0
)
```